server.js

```diff
@@ -57,9 +56,12 @@ app.prepare().then(() => {
       }
 
       // Custom route for all docs
-      app.render(req, res, '/doc', query)
+      app.render(req, res, '/doc', parseQuery(queryStr))
     } else {
-      // Regular Next.js handler
+      /*
+       * Regular Next.js handler
```

Comment:
I would actually use single line comments everywhere when it is a single line

Reply:
OK, updated.